src/main/java/org/assertj/core/internal/Iterables.java

```diff
@@ -1163,7 +1163,7 @@ public void assertContainsExactly(AssertionInfo info, Iterable<?> actual, Object
   }
 
   public <E> void assertSatisfiesExactly(AssertionInfo info, Iterable<? extends E> actual,
-                                         @SuppressWarnings("unchecked") Consumer<? super E>... allRequirements) {
+                                         Consumer<? super E>... allRequirements) {
```

Comment:
removing this leads to a "Type safety: Potential heap pollution via varargs parameter annotations" warning, adding @SafeVarargs is possible here since this is not proxyed or subclassed.

Reply:
Same comment as for the `Classes`